docs/guides/references/changelog.mdx

```diff
@@ -2,6 +2,14 @@
 title: Changelog
 ---
 
+## 13.14.0
+
+_Released 7/23/2024
+
+**Breaking Changes:**
+
+- Check of visibility is based on browsers checkVisibility. 'not.hidden' check is recommended before interaction with page elements now. 
+
```

Comment:
- There should be no Changelog entry added here. The Changelog text should only be added through https://github.com/cypress-io/cypress/pull/29741

```suggestion
```